pages/common/imapsync.md

```diff
@@ -0,0 +1,12 @@
+# imapsync
+
+> Email IMAP tool for syncing, copying and migrating email mailboxes
+> between two imap servers, one way, and without duplicates.
+
+- synchronize imap account between host1 and host2:
```

Comment:
As per the linter, descriptions should start with a capital letter.

Reply:
Fixed!